casper/src/main/scala/coop/rchain/casper/engine/LastFinalizedStateTupleSpaceRequester.scala

```diff
@@ -88,6 +89,9 @@ object LastFinalizedStateTupleSpaceRequester {
         requestQueue: Queue[F, Boolean]
     ): Stream[F, Boolean] = {
 
+      // Get importer
+      val importer = RSpaceStateManager[F].importer
+
       def broadcastStreams(ids: Seq[StatePartPath]) = {
```

Comment:
Not related to to this particular PR, but general observation. This `StatePartPath` made me a bit confused about its meaning. I get that this is a starting point from where chunk of `pageSize` is read and sent as a StoreItem.
Just a suggestion - maybe there is another name for that ? `TuplespacePointer`, `StoreItemStartHash`?

Reply:
It's definitely a path in the tree so it makes sense to have Path suffix. It's not path in the tuple space because tuple space is "encoded" on top of this tree.
_Start_ in the name can be misleading because state chunk have start and end path. Do you have some other suggestions? :)

NOTE: I'm saying a _tree_ and not a _trie_ because path is always referring to one _tree_ in a whole history _trie_.

Reply:
Ok, now its clear :) Though not completely. You are saying about end path - where is it encoded? I was thinking `StatePartPath` is start and pageSize defines end.
Would be great to have a description for this type https://github.com/rchain/rchain/blob/1f783b7762e46a7ef4fd78df8988307536df3090/casper/src/main/scala/coop/rchain/casper/engine/LastFinalizedStateTupleSpaceRequester.scala#L31
Why Is Seq here? Is it to enable to request multiple state chunks?

Reply:
This optional Byte is the _index_ of the child node, if trie node is the `PointerBlock`. It gives the path enough structure to index the whole trie in depth first traversal.
It's used as offset to position the child node.
https://github.com/rchain/rchain/blob/416936482a9632bf3732cfe7fcee46c603e028d3/rspace/src/main/scala/coop/rchain/rspace/state/RSpaceExporter.scala#L119-L122